static/docs/command-reference/remote/index.md

```diff
@@ -67,9 +67,7 @@ For the typical process to share the <abbr>project</abbr> via remote, see
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Examples
-
-1. Let's for simplicity add a default local remote:
+## Example: Add a default local remote:
```

Comment:
I would say that's not what we usually do in other place - we don't use second level headers to split steps. Rather we use them to split multiple examples.

Reply:
I don't think these are steps. I see these as totally separate/unrelated examples (except for sharing the name of the remote between the 2nd and 3rd ones). That's why I moved to the "Examples: ..." convention. They're pretty short examples, granted. Not sure they're worth putting here in the index even.

I could consolidate them into a single example that it a little more connected though. Would that be better?

Reply:
👍 k, makes sense. I was probably confused by the colons at the end of the titles. Probably we should remove those. I think it's fine to keep them separate (and revisit maybe at some point to make them more meaningful, but it applies to a lot of other DVC command reference examples.)

Reply:
Oh yeah definitely should remove those colons! Will do. Thanks for the heads up.

Reply:
Done.